src/main/java/org/craftercms/studio/api/v2/utils/StudioConfiguration.java

```diff
@@ -353,6 +353,8 @@
     // Content Service
     String CONTENT_ITEM_EDITABLE_TYPES = "studio.content.item.editableTypes";
 
+    String BLOB_INTERCEPTED_PATHS = "studio.blob.intercepted.paths";
```

Comment:
Is this needed?

Reply:
It should be there and used instead of magic strings